src/routes/(app)/namespaces/[namespace]/archival/+page.svelte

```diff
@@ -52,47 +53,43 @@
   {/if}
 {:else if archivalEnabled}
   <h1 class="text-2xl" data-testid="visibility-disabled-title">
-    This Namespace is currently enabled for archival but visibility is not
+    This namespace is currently enabled for archival but visibility is not
     enabled.
   </h1>
-  <p>To enable Visibility Archival:</p>
+  <p>To enable <a
+    class="text-blue-700 underline"
+    href="https://docs.temporal.io/clusters#archival"
+    target="_blank"
+    rel="noreferrer">archival visibility</a
+  >archival visibility</p>:</p>
```

Comment:
Looks like maybe some copy 🍝 made things a little wonky here.
```suggestion
  >:</p>
```